<p><b>[INFO]</b>&nbsp;POST Request : url=https://qa.api.zinnia.io/case/v1/tasks/message <br> path_url=/case/v1/tasks/message <br> headers={'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Type': 'application/json', 'Authorization': 'Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IlU1SWNUOEQ4LTNWdl80NDFIdnlzbyJ9.eyJodHRwczovL3FhLmFwaS56aW5uaWEuaW8vY2xpZW50SWRzIjpbImV2ZXJseSIsImVsaWMiLCJzYnVsIl0sImh0dHBzOi8vcWEuYXBpLnppbm5pYS5pby9wZXJtaXNzaW9ucyI6WyJyZWFkOnBvbGljeSIsIndyaXRlOnBvbGljeSIsIndyaXRlOmNhc2UiLCJyZWFkOmNhc2UiLCJyZWFkOmRvY3VtZW50Iiwid3JpdGU6ZG9jdW1lbnQiLCJ3cml0ZTp0YXNrIiwicmVhZDp0YXNrIl0sInBhcnR5SWQiOiJjNmE3YWI5ZjdkZDg0MDE1YTY1NTJiZmM1YjM2NmI3NyIsInNvdXJjZSI6IkJ1c2luZXNzIFByb2Nlc3MgTWFuYWdlbWVudCIsImlzcyI6Imh0dHBzOi8vbG9naW4ucWEuemlubmlhLmNvbS8iLCJzdWIiOiJVdU1yaWJyUDNlOHpWaGZIeTZ3UEoyNWxWWWgxQ1dOM0BjbGllbnRzIiwiYXVkIjoiaHR0cHM6Ly9xYS5hcGkuemlubmlhLmlvIiwiaWF0IjoxNzU4NjI2NzM2LCJleHAiOjE3NTg3MTMxMzYsInNjb3BlIjoib3BlbmlkIiwiZ3R5IjoiY2xpZW50LWNyZWRlbnRpYWxzIiwiYXpwIjoiVXVNcmliclAzZTh6VmhmSHk2d1BKMjVsVlloMUNXTjMifQ.Vt2WweG4lY6lNdA31PAvyzQ17BNp3Pvj4GF2qPd8-z5A4aSHht5BVJ0ZPDa9Niv7YBrnggmnZ-egKPEgMfZgvzndN5znI-a13yQNSB4x8dPiPsca363KfxoNFiK3MwbE--1r0oCLQ8IjXHa0hfiutLh7T4n-m8wuCFr5chbHCr6dBMHHDH3ZwTEZol4M5AE5HDXV6hoZ5UmMwdGYyg4260s-Ch6571TXK9W9E_Kcy4I8wu20Ob4e3bwVacnNXtB4Db3tzwwfhjiM8ABnV-D6YwDeDq_rBbhLIp0MB-ALOTw4_LanMdj0FgfAf0_YX3ObIyvCZrz4Rcc9FXLasu_GsA', 'Content-Length': '182'} <br> body=b'{"taskType": "SUITABILITY_REVI", "carrier": "MASS", "category": "SCHEDULED_REASON", "messageId": "DOCUMENT_MISSING", "message": "Document is missing for SUITABILITY_REVIEW for MASS"}' <br> </p><p><b>[INFO]</b>&nbsp;POST Response : url=https://qa.api.zinnia.io/case/v1/tasks/message <br> status=400, reason=Bad Request <br> headers={'Content-Type': 'application/json', 'Connection': 'keep-alive', 'x-correlation-id': '7ae97f21-2701-4548-8613-6374cedba4ab', 'Set-Cookie': 'AWSALBTG=c6dJOjpR2/HJAad9XBIx1YFdvvQeg2YIcYfJwFNUVwzxHLS6yWWpcE9FUTRIcx33Rnejj+pmOKSBXjFwq2i1X1ugC5Dh2B8+Wio5H9RwoD5rZN2072rb+g+3PUOwCqhIFbBRc9xmxOFLeHQagUt2DVqY9sy9Po6/nWu4FhWgtp3Y; Expires=Tue, 30 Sep 2025 11:25:37 GMT; Path=/, AWSALBTGCORS=c6dJOjpR2/HJAad9XBIx1YFdvvQeg2YIcYfJwFNUVwzxHLS6yWWpcE9FUTRIcx33Rnejj+pmOKSBXjFwq2i1X1ugC5Dh2B8+Wio5H9RwoD5rZN2072rb+g+3PUOwCqhIFbBRc9xmxOFLeHQagUt2DVqY9sy9Po6/nWu4FhWgtp3Y; Expires=Tue, 30 Sep 2025 11:25:37 GMT; Path=/; SameSite=None; Secure, AWSALB=etygBqbMTrjxy+iUivphhyuggFkKZ0LvFazsRROvxcLxsPK60YkgM0qQ1xckMpo39vFyRur1aBbCeLPLJ2BOpD4SzX+ZXFQRkmn/MkMeKYjOwNl/4EDoqZ0niOoY; Expires=Tue, 30 Sep 2025 11:25:37 GMT; Path=/, AWSALBCORS=etygBqbMTrjxy+iUivphhyuggFkKZ0LvFazsRROvxcLxsPK60YkgM0qQ1xckMpo39vFyRur1aBbCeLPLJ2BOpD4SzX+ZXFQRkmn/MkMeKYjOwNl/4EDoqZ0niOoY; Expires=Tue, 30 Sep 2025 11:25:37 GMT; Path=/; SameSite=None; Secure', 'X-Content-Type-Options': 'nosniff', 'Vary': 'Origin, Access-Control-Request-Method, Access-Control-Request-Headers', 'X-XSS-Protection': '1; mode=block', 'Expires': '0', 'X-Frame-Options': 'DENY', 'Content-Length': '206', 'Date': 'Tue, 23 Sep 2025 11:25:37 GMT', 'Pragma': 'no-cache', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Access-Control-Allow-Origin': '*', 'Access-Control-Expose-Headers': 'Host', 'X-Kong-Upstream-Latency': '14', 'X-Kong-Proxy-Latency': '45', 'Via': 'kong/3.6.1.6-enterprise-edition', 'X-Kong-Request-Id': '908c8c19c464cf58b7738e2ca680b778'} <br> body={"statusCode":400,"timestamp":"2025-09-23T11:25:37.548Z","errorId":"0e752338-cf22-44ea-8267-30414c1b8989","message":"A task message with the same taskType, carrier, category, and messageId already exists."} <br> </p><p><b>[INFO]</b>&nbsp;C:\Users\sharmash\AppData\Roaming\Python\Python312\site-packages\urllib3\connectionpool.py:1103: InsecureRequestWarning: Unverified HTTPS request is being made to host 'qa.api.zinnia.io'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings<br>  warnings.warn(</p><p style="color: red"><b>[FAIL]</b>&nbsp;HTTPError: 400 Client Error: Bad Request for url: https://qa.api.zinnia.io/case/v1/tasks/message</p>